doc/src/site/sphinx/tutorials/hive_support.rst

```diff
@@ -124,7 +125,7 @@ To run Sparkling Water with Hive support for kerberized hadoop cluster, you must
             conf.setHiveHost("hostname:10000") // The full address of HiveServer2
             conf.setHivePrincipal("hive/hostname@DOMAIN.COM") // Hiveserver2 Kerberos principal
             conf.setHiveJdbcUrlPattern("jdbc:hive2://{{host}}/;{{auth}}") // Doesn't have to be specified if host is set
-            val source = scala.io.Source.fromFile('hive.token')
+            val source = scala.io.Source.fromFile("hive.token")
```

Comment:
Fixes: ``/Users/kuba/devel/repos/sparkling-water/doc/src/site/sphinx/tutorials/hive_support.rst:119: WARNING: Could not lex literal_block as "scala". Highlighting skipped.``